scripts/decredStakeVoting.md

```diff
@@ -3,45 +3,44 @@
+`(ticket reward / ticket price) / ticket lifespan [in years] = Annual Percentage Rate`
 
-Stake voting requires no action by the stakeholder other than purchasing tickets. By default, tickets are configured to `abstain` from voting on current agenda items, however, `yes` or `no` votes can be configured easily through the Decrediton staking wallet.
+Assuming, for example, a ticket reward of 1.3 Decred, a ticket price of 88 Decred, with the average ticket lifespan of 30 days (or 0.082 years), the annual percentage rate of 12 tickets purchased in a year works out to 18%.
 
-Decrediton uses cryptographic ticket "receipts" to verify and allow stake voters to interface with Politeia; Decred's chain-anchored proposal system. Through Politeia, stake voters can participate in Decred self-governance by proposing issues for the community to vote on and voting on those proposed by others.
+Best of all, that 18% comes risk free; staked coins never leave your wallet and are always unlocked after a ticket expires, votes, or misses.
 
-By participating in Decred self-governance through the Stake Voting system, investors can earn significant returns.
+Stake voting is not only lucrative, but essential to the integrity of the Decred network as well. Here's how:
 
-This equation can be used to calculate the annual percentage rate of returns gained through proof-of-stake mining Decred.
+The 5 stake votes contained in each new block vote on the validity of the previous block. If this previous block is deemed invalid by at least a 3 to 5 majority, then all of its transactions and proof-of-work miner reward are stripped from the network while the voters' ticket rewards are retained. 
 
-`(ticket reward / ticket price) / ticket lifespan [in years] = Annual Percentage Rate`
+This process acts as a check on miner power, giving stakeholders a way to direct the rules of the network and prevent miners from misbehaving by withholding blocks or trying to modify the chain with a 51% attack of the network. Because every block is created by miners but requires stakeholder approval, the largest reorganization of the chain that can occur is just 2 blocks.
 
-Assuming, for example, a ticket reward of 1.3 Decred, a ticket price of 88 Decred, with the average ticket lifespan of 30 days (or 0.082 years), the annual percentage rate of 12 tickets purchased in a year works out to 18%.
+Tickets also vote concerning other agenda items up for a community vote such as any consensus rule upgrades being proposed. All of the code needed for the upgrade will already be built and ready for Decred stake holder approval. This was how the rules were upgraded to improve the efficiency of the ticket pricing algorithm, its how Decred will be upgraded to become compatible with lightning network, and it can also be used to change the size of Decred blocks, if the community desires.
```

Comment:
Should probably talking about this with the voting section before talking about rewards.